lib/vcr/test_frameworks/rspec.rb

```diff
@@ -8,7 +8,12 @@ module Metadata
       def configure!
         ::RSpec.configure do |config|
           vcr_cassette_name_for = lambda do |metadata|
-            description = metadata[:description]
+            description = if metadata[:description].empty?
+                            # we have an "it { is_expected.to be something }" block
+                            metadata[:scoped_id]
+                          else
+                            metadata[:description]
+                          end
```

Comment:
According to [this method](https://github.com/rspec/rspec-core/blob/19761e35a0fa8e7d61eaf4af52226fa0c2a39817/lib/rspec/core/metadata.rb#L173) in the rspec core, the description can never be nil.